internal/kubeclient/indexer_test.go

```diff
@@ -855,7 +855,7 @@ func TestIndexStagesByWarehouse(t *testing.T) {
 	}
 }
 
-func TestIndexServiceAccountsOIDCEmail(t *testing.T) {
+func TestIndexServiceAccountsOIDCClaimWithEmails(t *testing.T) {
```

Comment:
With the indexer in question now generically supporting any claim type, we probably don't need these claim-specific tests. Just one test of that indexer for any kind of claim should suffice.